lighthouse-core/audits/installable-manifest.js

```diff
@@ -51,59 +138,82 @@ class InstallableManifest extends MultiCheckAudit {
   }
 
   /**
-   * @param {LH.Artifacts.ManifestValues} manifestValues
-   * @return {Array<string>}
+   * @param {LH.Artifacts} artifacts
+   * @return {Array<LH.IcuMessage>}
    */
-  static assessManifest(manifestValues) {
-    if (manifestValues.isParseFailure && manifestValues.parseFailureReason) {
-      return [manifestValues.parseFailureReason];
+  static getInstallabilityErrors(artifacts) {
+    const installabilityErrors = artifacts.InstallabilityErrors.errors;
+    const errorMessages = [];
+
+    // Filter out errorId 'in-incognito' since Lighthouse users are recommended to use incognito to test.
+    installabilityErrors.filter(err => err.errorId !== 'in-incognito');
+
+    for (const err of installabilityErrors) {
+      let matchingString;
+      try {
+        // @ts-expect-error errorIds from protocol should match up against the strings dict
+        matchingString = UIStrings[err.errorId];
```

Comment:
This line is a potential typescript error, but it won't ever throw at runtime (it'll just be `undefined`). Can just do an `if (matchingString === undefined) {` check (though the compiler may complain about that too)

Reply:
> This line is a potential typescript error, but it won't ever throw at runtime (it'll just be `undefined`). Can just do an `if (matchingString === undefined) {` check (though the compiler may complain about that too)

this doesn't seem like it got changed?